src/main/java/Greet.java

```diff
@@ -0,0 +1,79 @@
+public class Greet {
+    private static final int MAX_ARRAY_LIMIT = 100;
+    protected static Task[] list = new Task[MAX_ARRAY_LIMIT];
+    private static int tasksAdded = 0;
+
+    public static void printLineOnConsole() {
+        System.out.println("    ____________________________________________________________");
+    }
+
+    /**
+     * Add new Task in list
+     *
+     * @param newTask Task object created based on command.
+     */
+    //might move to main/duke class
+    public static void addTask(Task newTask) {
+        list[tasksAdded] = newTask;
+        tasksAdded++;
+        printAddNewTask(newTask);
+    }
+
+    /**
+     * Calls Task.markAsDone().
+     *
+     * @param taskNumber Task number that is tagged to the task on console.
+     */
+    public static void checkDoneTask(int taskNumber) {
+        int taskIndex = taskNumber - 1;
+        list[taskIndex].markAsDone();
+    }
```

Comment:
The naming of this method may not be the clearest (I misinterpreted it as a boolean to check if a particular task is done). Also perhaps maybe you could have list[taskNumber - 1].markAsDone() instead of calling a new method for it.